ansible-remove-cluster/roles/openstack-cluster-hosts/tasks/main.yaml

```diff
@@ -0,0 +1,39 @@
+- name: Get cloud configuration
+  tags: bootstrap
+  os_client_config:
+
+- set_fact:
+    deploymentkeyisset: "{{ lookup('env','KEY') }}"
+    deploymentkeyname: "{{ name }}-key"
+
+- set_fact:
+    deploymentkeyname: "{{ lookup('env','KEY') }}"
+  when: deploymentkeyisset != ""
+
+- name: Delete cluster hosts - master
+  os_server:
+    security_groups:  "{{ lookup('env','SECURITY_GROUP_CLUSTER') | default('sg-cluster', true) }},{{ lookup('env','SECURITY_GROUP_GLUSTER') | default('sg-gluster', true) }},galaxy"
```

Comment:
In theory we should be able to get away w/o the `galaxy` SG anyway, was a copy-and-paste remnant. The SG you created was empty?

Reply:
Yes, I did copy-paste those SGs.